tests/jenkins/.gitignore

```diff
@@ -0,0 +1,7 @@
+__pycache__
```

Comment:
Are you suggesting that I remove the entire file as it is redundant, or just certain lines?